README.md

```diff
@@ -666,7 +666,8 @@ app.controller('MainCtrl', function(Restangular, BingRestangular) {
 
 ### Decoupled Restangular Service
 
-There're some times where you want to use Restangular but you don't want to expose Restangular object anywhere. For those cases, you can actually use the `service` feature of Restangular.
+There're some times where you want to use Restangular but you don't want to expose Restangular object anywhere. 
```

Comment:
*There are

I know this is in the original docs, but would you mind fixing this sorta typo while you're here?